jobs/scripts/nightly-samba-builds/nightly-samba-builds.sh

```diff
@@ -72,14 +72,8 @@ then
 
 fi
 
-if [[ "${PLATFORM}" = "fedora" ]]
-then
-	make "rpms.fedora" "vers=${VERSION}" "refspec=${SAMBA_BRANCH}"
-	make "test.rpms.fedora" "vers=${VERSION}" "refspec=${SAMBA_BRANCH}"
-else
-	make "rpms.${PLATFORM}${VERSION}" "refspec=${SAMBA_BRANCH}"
-	make "test.rpms.${PLATFORM}${VERSION}" "refspec=${SAMBA_BRANCH}"
-fi
+make "rpms.${PLATFORM}" "vers=${VERSION}" "refspec=${SAMBA_BRANCH}"
+make "test.rpms.${PLATFORM}" "vers=${VERSION}" "refspec=${SAMBA_BRANCH}"
```

Comment:
I was expecting to see `make rpms.${PLARFORM}` **instead of** an if/else  branching.
Doesn't keeping the if/else branch defeat the purpose of this PR and the preparatory samba-build PR?

I suggest to remove the if-else branch and only use `make rpms.${PLATFORM}` directly.

Reply:
I was not looking right, confused removal with addition. forget this review...